Cannot compile quickwit on arch different from `x86_64`

```bash
error[E0599]: no variant or associated item named `AVX2` found for enum `FilterImplPerInstructionSet` in the current scope
  --> /Users/fmassot/.cargo/git/checkouts/tantivy-f70b7ea03dadae9a/059fc76/bitpacker/src/filter_vec/mod.rs:38:49
   |
11 | enum FilterImplPerInstructionSet {
   | -------------------------------- variant or associated item `AVX2` not found for this enum
...
38 |         if code == FilterImplPerInstructionSet::AVX2 as u8 {
   |                                                 ^^^^ variant or associated item not found in `FilterImplPerInstructionSet`

```

--- a/quickwit/Cargo.toml
+++ b/quickwit/Cargo.toml
@@ -219,7 +219,7 @@ quickwit-serve = { version = "0.5.0", path = "./quickwit-serve" }
 quickwit-storage = { version = "0.5.0", path = "./quickwit-storage" }
 quickwit-telemetry = { version = "0.5.0", path = "./quickwit-telemetry" }
 
-tantivy = { git = "https://github.com/quickwit-oss/tantivy/", rev = "059fc767e", default-features = false, features = [
+tantivy = { git = "https://github.com/quickwit-oss/tantivy/", rev = "057211c3", default-features = false, features = [
   "mmap",
   "lz4-compression",
   "zstd-compression",

